Update product-management-team-handbook.md (#347)
mattermost/mattermost-handbook: operations/research-and-development/product/product-management-team-handbook.md

* * What fundamental philosophies are predominant?
  * Pragmatic Marketing: “Your opinion, although interesting, is irrelevant.”
    * It’s all about the customer/user/buyer’s opinion/want/need

> * FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\)

Reviewer: Could change the link to: https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq.

> * FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\)

Author: ```suggestion
* FAQ \([https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\)
```

> * FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\

Reviewer: ```suggestion
* FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)
```

> * FAQ \([https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\)

Reviewer: ```suggestion
* FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\
```

* Make your first edit and pull request on GitHub
  * Update documentation or product help text
* Small contributions
  * File a bug in Jira
  * Test a new feature on a test server, and suggest an improvement
  * Leave feedback on a design proposal

Readings:

* The Advantage:
  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
* The Five Dysfunctions of a Team: A Leadership Fable
  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)

### First 30 Days

Objective: Build foundational knowledge

Training focus:

* Users and customers
* Product
  * Features/Functions
  * Technical
* Market and Competitive Landscape
* Team
  * People and Processes

Activities:

* Learn about our customers and users
  * Support
    * Customers: Zendesk \(Sven\)
      * Meet with Sven for Zendesk training and support overview
      * Join support standups for a week or two, and help out on support tickets
    * Community: GitHub, Forums \(Amy\)

>   * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\)

Reviewer 2: ```suggestion
  * Customer success \(Meet with a CSM - Jennifer Lawler, Jeff Johnson, Sasa Cosic, Rosie Lane\)
```

    * [Process feedback in productboard](https://mattermost.productboard.com/insights/shared-inbox) 
  * Learn about the market
  * Sales \(Regional VPs and Customer Engineers\)
    * Sales team overview
    * Sales team priorities
    * What competitors are we up against
    * How does Sales team position the product
    * What are typical challenges for the Sales team
    * Which sales do we win, which do we lose
    * What are the most common questions on configurations and trials
  * Marketing \(Marketing team lead\)
    * Marketing team overview
    * What are the Marketing team’s priorities
    * What is our current messaging and positioning around Mattermost
    * What is needed from Product Managers
  * Mini competitive research projects
    * Review of MS Teams, Slack, Discord, and other chat products
* Spend a week doing customer support
  * Review documentation first and work with Support Lead or PM mentor to answer any questions documentation does not address
  * Update docs with answers as appropriate
* Spend a couple weeks joining as many customer and prospect calls as possible

>   * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)

Reviewer: ```suggestion
  * Take detailed notes and post to [https://community.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```

>   * Take detailed notes and post to [https://community.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)

Author: ```suggestion
  * Take detailed notes and log in [Salesforce](https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce)
```

    * Dive into product analytics
* Learn about the product development process
  * High level overview \(PM buddy\)
    * Team structure
    * [Development process overview](https://docs.mattermost.com/process/overview.html)
  * Team specific processes \(PM buddy\)
    * Team Triage
    * Sprint Planning
    * OKRs
    * Join general Triage meeting
    * Join all feature team meetings \(Triage, Sprint Planning\)
  * Design process \(Feature team designer\)
    * [Spec template](https://docs.google.com/document/u/1/d/1QC4-qysnp3DuPS5nROUpL31WtiJ7RhZCXz24T1WgTko/edit#heading=h.2msta3abogxt)
      * Competitor research
      * User research
    * Review process
      * Spec review channel
  * Working with contributor community \(Jason\)
    * Help wanted tickets
    * Community buddies
    * …?
  * Release process
    * QA process \(QA team member\)
      * General QA process
      * Sites for testing
      * Release testing process
        * Add to release testing
        * [Release Testing assignment example](https://docs.google.com/spreadsheets/d/1gW7i8uF3jd5wuHd5mJ7UdBQsMFVZwHKMXE7WOtlDa0k/edit#gid=1130032476)
    * Release process \(Amy, Release Manager\)

>       * [Release process](https://docs.mattermost.com/guides/core.html#release-process)

Reviewer: ```suggestion
      * [Release process](https://handbook.mattermost.com/operations/research-and-development/product/release-process)
```

A good place to start is exploring the websites, and then see if you can find a way to install a version of Mattermost to try it out.

* Google Mattermost
* [https://mattermost.com/](https://mattermost.com/)
* [https://docs.mattermost.com/](https://docs.mattermost.com/) \(User, Admin, Integrations guides\)
* [https://developers.mattermost.com/](https://developers.mattermost.com/)
* [http://github.com/mattermost](http://github.com/mattermost)
* [https://www.mattermost.com/demo](https://www.mattermost.com/demo)

Activities:

> * Meet with Lindsay

Reviewer 3: Can change to meet w/ manager

> * Meet with Lindsay

Author: ```suggestion
* Meet with Manager
```

Customer Obsession:

* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week:
  * [CSM:PM Engagement](https://community-daily.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Emily, Rosie, Jenn\)
  * [Solution Architects](https://community-daily.mattermost.com/private-core/channels/solutions-architects)
  * [Pre-Sales Support](https://community-daily.mattermost.com/private-core/channels/pre-sales-support)
* Join and review the topics and posts in these channels to get a better understanding of our customers
  * [Customer Feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
  * [Customer Support](https://community-daily.mattermost.com/private-core/channels/community)
* Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
* Begin reading [docs.mattermost.com](https://docs.mattermost.com/)

Team:

* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
  * Ask questions to understand development, design, QA processes
* Meet with Amy \(Release Manager\) and read the release process doc

> * Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you

Author: This link is a 404. Confirm correct page is https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours.

Reviewer 3: it should be - https://github.com/it33/readme/blob/master/README.md for Ian's readme file, and I *think* the "CEO Informational Meeting" linked in the readme is the first meeting people have (you can confirm with Amy Nicol if that's the right meeting).

Author: Thanks, @lfbrock. 

@amynicol1985, in the context of meeting with Ian for the first time is the link provided aimed at a specific meeting outline or just as an overview of Ian ahead of time?

Author: Will follow this up post-merge as it's not a blocker.

> * Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you

Author: ```suggestion
* Meet with Ian \([https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you
```

  * Get Jira training from PM buddy
    * [Customer Requests](https://mattermost.atlassian.net/projects/CR/issues/CR-16?filter=allopenissues)
    * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
* Make your first edit and pull request on GitHub
  * Update documentation or product help text
* Small contributions
  * File a bug in Jira
  * Test a new feature on a test server, and suggest an improvement
  * Leave feedback on a design proposal

Readings:

* The Advantage:
  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
* The Five Dysfunctions of a Team: A Leadership Fable
  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)

### First 30 Days

Objective: Build foundational knowledge

Training focus:

* Users and customers
* Product
  * Features/Functions
  * Technical
* Market and Competitive Landscape
* Team
  * People and Processes

Activities:

* Learn about our customers and users
  * Support

>     * Customers: Zendesk \(Paul\)

Author: Check whether there's a new DRI for this.

Reviewer 3: @justinegeffen it's Sven now

      * Meet with Sven for Zendesk training and support overview
      * Join support standups for a week or two, and help out on support tickets
    * Community: GitHub, Forums \(Amy\)
  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\)
    * Overview of Customer Success team, who owns what
    * Current Customer Success processes
    * Typical challenges during customer onboarding
    * Join one CSM team meeting to meet the rest of the CS team
  * Customer calls
    * Join calls with CSMs
    * Join calls with Sales team
  * Feature requests
    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)

>     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 

Reviewer 3: We should replace this link with "Process feedback in productboard" with a link to productboard inbox: https://mattermost.productboard.com/insights/shared-inbox

      * Track down estimates and early dev feasibility discussions on [Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
* Learn about the market

>   * Sales \(Sales - Steve, Lance, Phillip and Solution Architects - Mark, John, Christian\)

Author: Confirm correct department/roles are listed.

Reviewer 3: @justinegeffen maybe we can remove the names for now and people can check in Bamboo

Reviewer 3: Actually looking below -- Maybe using titles is better. We can saw "Regional VPs and Customer Engineers"

>       * Track down estimates and early dev feasibility discussions on [Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)

Reviewer 3: remove this link, we use productboard now

    * Sales team overview
    * Sales team priorities
    * What competitors are we up against in sales
    * How does Sales team position the product
    * What are typical challenges in sales
    * Which sales do we win, which do we lose
    * What are the most common questions on configurations and trials

>   * Marketing \(Marketing team lead\)

Author: Confirm whether this is "Marketing team lead" or "Kendall - Marketing Operations Manager".

Reviewer 3: Let's leave as generic titles, I like that better as it's easier to keep up to date. We can point them to the right people when we onboard them.

    * Marketing team overview
    * What are the Marketing team’s priorities
    * What is our current messaging and positioning around Mattermost
    * What is needed from Product Managers
  * Mini competitive research projects
    * Review of MS Teams, Slack, Discord, and other chat products
* Spend a week doing customer support
  * Review documentation first and work with Support Lead or PM mentor to answer any questions documentation does not address
  * Update docs with answers as appropriate
* Spend a couple weeks joining as many customer and prospect calls as possible

>   * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)

Reviewer: ```suggestion
  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```

>   * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)

Reviewer 3: Notes should be filed in Salesforce now, Emily Hollinger has the link to the new process (Salesforce auto-posts to the channel)

Author: @emilyhollinger, would you be open to sharing the link to the process, or details that would work in this context?

Reviewer 2: @justinegeffen here's the link: https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce

* Learn about the product

>   * Product Direction \(Lindsay, Product Lead\)

Reviewer 3: Can change to manager here as well

    * [Company VPMOM](https://handbook.mattermost.com/operations/operations)
    * [R&D V2MOM](https://docs.google.com/document/d/18tK7XmW2fGaUP2otiItHKcHyMGUf3kgnZxOQM6WB4zM/edit)
    * Public Roadmap \(Staff Team &gt; [Roadmap channel](https://community-daily.mattermost.com/private-core/channels/roadmap)\)
    * Review videos on [Mattermost Academy](https://academy.mattermost.com/)
  * Technical Architecture \(Feature team dev lead\)
    * Reference:
      * [High level overview \(slideshow summary\)](https://docs.google.com/presentation/u/1/d/1Z44Yhaab7ul7yrnyYjQpq_T2A_z0GbrYMYe8Ov6zRZw/edit#slide=id.g1994d3cd93_0_507)
      * [High level system design document \(more detailed\)](https://docs.google.com/document/d/1NF6D3K83FfDaVcbPNeIFhS-Uw-otQEcsSecxQZhOaEQ/edit)
      * [Navigating the Mattermost database](https://docs.google.com/document/d/1BuZaYHkmUgitKtLNbeV3mq6xa0ZwFyPw8Qc6aFZalZE/edit?ts=5cc0bb36)
      * Typical customer deployment
        * Sizing recommendations

>   * Product areas: Functional Overview and Roadmap

Author: Confirm areas are correct.

Reviewer 3: We can use the product pillars outlined by Chandar: 
Cloud - Adam Clarkson
End user experience (Desktop, Web, and Mobile client applications) - Eric Sethna
Enterprise-grade (Platform, Admin Console) - Dennis Kittrell
Product-centric GTM (Customer Portal, Customer Journey) - Eric Sadur
App Marketplace & Developer Toolkit - Aaron Rothschild 
Workflows - Ian Tao 
Community - Jason Blais

    * End user experience (\Desktop, Web, and Mobile client applications\) \(Eric Sethna\)
    * Cloud \(Adam Clarkson\)
    * App Marketplace and Developer Toolkit \(Aaron Rothschild\)
    * Enterprise-grade \(Platform, Admin Console\) \(Dennis Kittrell\)
    * Product-centric GTM \(Customer Portal, Customer Journey\) \(Eric Sadur\)
    * Workflows \(Ian Tao\)
    * Community \(Jason Blais\)
    * Reference:

>       * [Roadmap - All teams](https://docs.google.com/spreadsheets/u/1/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit#gid=0)

Reviewer 3: ```
- [Roadmap - All teams](https://mattermost.productboard.com/portal/19-combined-highlighted-features)
- [Release Plan](https://mattermost.productboard.com/feature-board/1749949-release-summary-internal)
```

* Capture SWAG estimates from design and developers to determine length project
* Facilitate prioritization decisions in sprint and release planning meetings
* Keep [Release Plan Roadmap](https://docs.google.com/spreadsheets/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit?usp=sharing) up to date with features queued for the release

>   * Prepare updates for the

Author: Missing a link/sentence.

Reviewer 3: Not sure what this is, maybe we can remove it if Katie doesn't know either

Author: Removed for now.

#### Feature Design

* Define requirements and coordinate design resources for features
* Prepare and manage spec documents to organize planning, design, and stakeholder input of features
* Ensure cross-collaboration between designers, developers, and QA
* Create Jira tickets for features and bugs
* More complete information on the Design process can be found [here](https://docs.google.com/document/d/1PMGeoh95CyLZ2lfAjOh6p2HuFrysx5HXit8XoOrm8Pg/edit)
* Ensure designs follow our [UX guidelines](https://docs.mattermost.com/developer/fx-guidelines.html#fast-obvious-forgiving)
  * Fast, Obvious, Forgiving
* Determine feature availability: Team Edition or Enterprise Edition
  * [stewardship principles](https://docs.mattermost.com/developer/manifesto.html#stewardship-principles-for-the-mattermost-open-source-project) that we use to guide whether a feature belongs in TE

>   * [TE vs EE from Ian](https://pre-release.mattermost.com/private-core/pl/m1o63kk3h7numxw3z116ft4ffw)

Reviewer: ```suggestion
  * [TE vs EE from Ian](https://community-release.mattermost.com/private-core/pl/m1o63kk3h7numxw3z116ft4ffw)
```

* Capture SWAG estimates from design and developers to determine length project
* Facilitate prioritization decisions in sprint and release planning meetings

> * Keep [Release Plan Roadmap](https://docs.google.com/spreadsheets/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit?usp=sharing) up to date with features queued for the release

Reviewer 3: We use Productboard for this now instead of google spreadsheet

Reviewer 3: `[Release Plan](https://mattermost.productboard.com/feature-board/1097526-release-tracking-internal)`

* Create Jira tickets for features and bugs
* More complete information on the Design process can be found [here](https://docs.google.com/document/d/1PMGeoh95CyLZ2lfAjOh6p2HuFrysx5HXit8XoOrm8Pg/edit)
* Ensure designs follow our [UX guidelines](https://docs.mattermost.com/developer/fx-guidelines.html#fast-obvious-forgiving)
  * Fast, Obvious, Forgiving
* Determine feature availability: Team Edition or Enterprise Edition
  * [stewardship principles](https://docs.mattermost.com/developer/manifesto.html#stewardship-principles-for-the-mattermost-open-source-project) that we use to guide whether a feature belongs in TE
  * [TE vs EE from Ian](https://community-release.mattermost.com/private-core/pl/m1o63kk3h7numxw3z116ft4ffw)
* Determine Packaging/SKU \(E10 & E20\)
  * TE OR EE
    * [https://docs.mattermost.com/overview/product.html\#mattermost-editions](https://docs.mattermost.com/overview/product.html#mattermost-editions) 
  * In-Code: if \(this.props.buildEnterpriseReady && this.props.isLicensed
  * Review with PM team and get approval from other leads as required

### Feature Development and The Release Process

#### Triage

> * Participate in 2x weekly Triage meetings \(may be daily towards end of release\)

Author: ```suggestion
* Participate in daily Triage meetings \(not mandatory to join every meeting\)
```

  * In-Code: if \(this.props.buildEnterpriseReady && this.props.isLicensed
  * Review with PM team and get approval from other leads as required

### Feature Development and The Release Process

#### Triage

> * Participate in 2x weekly Triage meetings \(may be daily towards end of release\)

Reviewer: These are currently daily meetings. However, not mandatory for all PMs to join every day.

    * Any in product instructions or documentation should never contain hard-coded links
    * A redirect page from about.mattermost.com should be used in product instructions. \(See instructions [here](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#how-do-i-create-redirects-for-in-product-documentation)\)
* Maintain product accuracy on mattermost.com/product and roadmap
* Author [Forum](https://forum.mattermost.org/t/community-design-meeting-folded-reply-threads/6729/10), [blog](https://mattermost.com/blog/mattermost-launches-mattermost-superstars-program-for-contributors/) and [release announcement](https://mattermost.com/blog/mattermost-release-v5-25/) posts
  * Example \(link to Eric’s permissions post\)
* Other Marketing \(eg; future webinars, event participation, etc\)

### Customer and Community Success

#### Customer Success

* Provide Roadmap status updates for Enterprise clients
* Assist with Customer Support - Tier 2 on call weekly rotation
* PMs should be on sales/customer calls often; if not on at least a couple per week then request to be added in the CSM:PM channel and Pre-sales channel

>   * Document and post feedback via the [Customer Feedback channel](https://community.mattermost.com/private-core/channels/customer-feedback)

Author: ```suggestion
  * Document and post feedback in [Salesforce](https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce)
```

    * Any in product instructions or documentation should never contain hard-coded links.
    * A redirect page from about.mattermost.com should be used in product instructions. \(See instructions [here](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#how-do-i-create-redirects-for-in-product-documentation)\)
* Maintain product accuracy on mattermost.com/product and roadmap
* Author [Forum](https://forum.mattermost.org/t/community-design-meeting-folded-reply-threads/6729/10), [blog](https://mattermost.com/blog/mattermost-launches-mattermost-superstars-program-for-contributors/) and [release announcement](https://mattermost.com/blog/mattermost-release-v5-25/) posts
  * Example \(link to Eric’s permissions post\)
* Other Marketing \(eg; future webinars, event participation, etc\)

### Customer and Community Success

#### Customer Success

* Provide Roadmap status updates for Enterprise clients
* Assist with Customer Support - Tier 2 on call weekly rotation
* PMs should be on sales/customer calls often; if not on at least a couple per week then request to be added in the CSM:PM channel and Pre-sales channel

>   * Document and post feedback via the [Customer Feedback channel](https://community.mattermost.com/private-core/channels/customer-feedback)

Reviewer 3: Update link here as well to be filing in Salesforce w/ Emily's new process

Please encourage colleagues to file feedback in productboard when they are discussing requests in Mattermost channels, so that we have single source with all requests. PMs are responsible for reviewing the productboard feedback, and assigning it to relevant feature ideas. As they are reviewing the requests, PMs may also:

* Work with Customer Success and Sales to clarify use cases, root issues, and pain points as well as the priority of the request to the customer
* Propose alternate solutions and workarounds when they are available
* Provide feedback on whether it's a "Won't Do" feature that we are unlikely to add to the product
* Update the feature idea to link any implementation or design tickets
* Update the **Effort** for the feature with a SWAG from the Engineering team
* Update the feature idea to have a release timeframe (3 months, 6 months, future) or release version if the feature is planned

After adding the feedback to the relevant idea in productboard, the PM comments and mentions the reporter so they are aware the feedback has been processed, and then marks the item as **Processed**.

#### Customer Request Rotation

> In sync with the Twitter rotation, Product Managers rotate responsibility weekly for monitoring newly-filed [customer requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97&issueLimit=100) and ensuring newly filed requests are assigned to the appropriate PM according to areas of ownership. The weekly PM meeting is the time to identify requests where ownership is unlcear or merits discussion. The PM on rotation should also remind fellow PMs in the Product Management channel to review their requests and at a minimum update the Fix Version.

Reviewer 3: we can remove this section, each PM now processes requests in Productboard so we no longer have the customer request rotation

Author: Removed, thanks. :)

* Maintain product accuracy on mattermost.com/product and roadmap
* Author Forum and blog posts
  * Example \(link to Eric’s permissions post\)
* Other Marketing \(eg; future webinars, event participation, etc\)

### Customer and Community Success

#### Customer Success

* Provide Roadmap status updates for Enterprise clients
* Assist with Customer Support - Tier 2 on call weekly rotation
* PMs should be on sales/customer calls often; if not on at least a couple per week then request to be added in the CSM:PM channel and Pre-sales channel
  * Document and post feedback via the [Customer Feedback channel](https://community.mattermost.com/private-core/channels/customer-feedback
)
  * Add/update Jira tickets to capture feature requests
* Participate in and contribute updates via the Customer Request Triage meeting
* Verify bugs reported by customers
* Answer customer questions in Premier Support \(PS: Customer name\) Channels and in internal channels by CSM and Sales

#### Customer Requests

> Feature and improvement requests from our Enterprise customers are logged by Sales Architects, Customer Success, and Support in a [Customer Request Jira project](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&issueLimit=100).

Reviewer 3: These are logged in Productboard now. And should be Customer Engineers instead of Sales Architects.

> Please encourage collegues to add tickets when they are discussing requests in Mattermost channels, so that we have single source with all requests. Product Managers are responsible for reviewing Customer Requests regarding features and product improvements. New tickets may already be assigned to a PM based on area of responsibility. Please review tickets often to address newly created tickets and to update any older tickets as new information becomes available. For each ticket or request:

Reviewer 3: This whole section below should be updated to something like: 

Please encourage colleagues to file feedback in Productboard when they are discussing requests in Mattermost channels, so that we have single source with all requests. PMs are responsible for reviewing the Productboard feedback, and assigning it to relevant feature ideas. As they are reviewing the requests, PMs may also: 
- Work with Customer Success and Sales to clarify use cases, root issues and pain points, and the priority of the request to the customer
- Propose alternate solutions and workarounds when they are available
- Provide feedback on if it is a "Won't Do" feature that we are unlikely to add to the product 
- Update the feature idea to link any implementation or design tickets
- Update the "Effort" for the feature with a SWAG from the engineering team
- Update the feature idea to have a release timeframe (3 months, 6 months, future) or release version if the feature is planned 

After adding the feedback to the relevant idea in Productboard, the PM comments and mentions the reporter so they are aware the feedback has been processed, and then marks the item as "Processed".

#### Documentation

> * * Update/create documentation for features/PRs

Reviewer: ```suggestion
* Update/create documentation for features/PRs
```

  * Maintain documentation on feature proposals \(reference all design and development documentation\)
  * Release notes should be written targeting an international IT Admin
  * Guidelines for [documentation](https://handbook.mattermost.com/operations/research-and-development/product/product-technical-writing-team-handbook#submitting-feature-documentation-pms)
    * Any in product instructions or documentation should never contain hard-coded links.
    * A redirect page from about.mattermost.com should be used in product instructions. \(See instructions [here](https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#how-do-i-create-redirects-for-in-product-documentation)\)
* Maintain product accuracy on mattermost.com/product and roadmap
* Author Forum and blog posts

>   * Example \(link to Eric’s permissions post\)

Author: Confirm which post to link.

Reviewer 3: Not sure, maybe use these examples? 
Author [forum](https://forum.mattermost.org/t/community-design-meeting-folded-reply-threads/6729/10), [blog](https://mattermost.com/blog/mattermost-launches-mattermost-superstars-program-for-contributors/), and [release announcement](https://mattermost.com/blog/mattermost-release-v5-25/) posts 

Or @wiersgallak any thoughts on what would be good examples here?

Author: I've updated with your suggestions and happy to iterate in a new PR. :)

#### Twitter Rotation

> Product Managers rotate responsibility weekly for monitoring and responding to Mattermost mentions on our corporate Twitter account. The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on the first day of their rotation. The PM that was previously on rotation should reach out to the new PM on rotation to hand off the duties and to remind on the additional responsibilities of assigning unassigned Customer Requests. Below are the responsibilities of the PM on Twitter rotation:

Reviewer 3: Change to 
"Product Managers rotate responsibility weekly for answering product question escalations on Twitter. Twitter is monitored by the marketing team, and they will @mention the PM on rotation in the Twitter channel when they need to escalate a question. 

The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community-daily.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on the first day of their rotation."

> * Monitor Twitter channel in Mattermost Contributors Team: [https://community.mattermost.com/core/channels/twitter](https://community.mattermost.com/core/channels/twitter)

Reviewer 3: We can remove all these bullet points, up to and including the assign any unassigned customer requests

    </tr>
  </thead>
  <tbody></tbody>
</table>

> ### Monthly Roadmap Q&A - Pre-sales and Support

Reviewer 3: This meeting is now: 
- Monthly Product:Sales Call
- 2nd Monday of the month, 9am to 10am PST 
- Attendees: All PMs; Optional: Sales, BizOps, Marketing departments 
- Format: 
  - PMs share roadmap highlights
    - Focus is on features with clear release timelines, and a clear benefit to customers, so Sales knows what is coming up soon they can talk to customers about 
    -  Demos, PRFAQs, and designs are shared as needed to illustrate the benefits
  - Open Q&A

To officially release a feature, it should meet the following criteria:

1. **Scalable:** Works for 60k concurrent connections, 60M rows in the database, and in High Availability
2. **Cross Platform:** Works on all web view sizes, mobile apps, desktop apps
3. **Measurable:** Telemetry added where appropriate
4. **Documented:** Complete documentation is available
5. **Secure:** Reviewed internally for security, and included in future security reviews

> 6. **Accessible:**: Accessibility has been added for user interface components

Reviewer: ```suggestion
6. **Accessible:** Accessibility has been added for user interface components
```

7. **Logged:** Audit log entries have been added to allow system admins to troubleshoot issues

> 8. **Compliant:**: Anything needed for compliance is tracked in logs

Reviewer: ```suggestion
8. **Compliant:** Anything needed for compliance is tracked in logs
```
